bin/update-dockerfiles.sh

```diff
@@ -10,16 +12,15 @@ update_musl_linux() {
     CORRETTO_VERSION=$1
     MAJOR_RELEASE=$2
 
-    sed -i "" "s/ARG version=.*/ARG version=${CORRETTO_VERSION}/g" ./${MAJOR_RELEASE}/jdk/alpine/Dockerfile
+    ${SED} "s/ARG version=.*/ARG version=${CORRETTO_VERSION}/g" ./${MAJOR_RELEASE}/jdk/alpine/Dockerfile
```

Comment:
I think we should create a backlog item to change this to use templates and not a ton of sed commands.

Reply:
Created Issue #81 for this.